src/crypto/backup.ts

```diff
@@ -41,6 +40,7 @@ import { CryptoEvent } from "./index.ts";
 import { ClientPrefix, HTTPError, MatrixError, Method } from "../http-api/index.ts";
 import { BackupTrustInfo } from "../crypto-api/keybackup.ts";
 import { BackupDecryptor } from "../common-crypto/CryptoBackend.ts";
+import { encodeRecoveryKey } from "../crypto-api";
```

Comment:
```suggestion
import { encodeRecoveryKey } from "../crypto-api/index.ts";
```

Otherwise you will re-introduce https://github.com/matrix-org/matrix-js-sdk/issues/4287